init_process.go

```diff
@@ -0,0 +1,406 @@
+package main
+
+import (
+	"encoding/json"
+	"io/ioutil"
+	"log"
+	"net/http"
+	"os"
+	"reflect"
+	"regexp"
+	"strings"
+
+	"github.com/aws/aws-sdk-go/aws"
+	"github.com/aws/aws-sdk-go/aws/arn"
+	"github.com/aws/aws-sdk-go/aws/session"
+	"github.com/aws/aws-sdk-go/service/s3"
+	"github.com/aws/aws-sdk-go/service/s3/s3manager"
+)
+
+// static file path
+const ConfigFileFolderPath = "init_process_fluent-bit_s3_config_files"
+const MainConfigFilePath = "init_process_fluent-bit.conf"
+const OriginalMainConfigFilePath = "/fluent-bit/etc/fluent-bit.conf"
+const InvokerFilePath = "invoker.sh"
+
+// default Fluent Bit command
+var FluentBitCommand = "exec /fluent-bit/bin/fluent-bit -e /fluent-bit/firehose.so -e /fluent-bit/cloudwatch.so -e /fluent-bit/kinesis.so"
```

Comment:
`fluentBitCommand`

Reply:
just for consistency: https://github.com/aws/amazon-cloudwatch-logs-for-fluent-bit/blob/mainline/cloudwatch/cloudwatch_test.go

Reply:
Already using the consistent style